recipes/g2clib/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+    name: g2clib
+    version: 1.5.0
+
+source:
+    fn: g2clib-1.5.0-patch.tar.gz
+    url: http://www.ncl.ucar.edu/Download/files/g2clib-1.5.0-patch.tar.gz
+    md5: f857f49a66918316996eb52399e04120
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - libpng
+        - zlib
+        - jpeg
+        - jasper
+        - gcc
```

Comment:
Ok, there are a few in this PR like this one. Let's see if we can't drop them. I know I saw some issues with `libiconv` related to this so we should see if we can't just avoid Continuum's `gcc` and use system compilers instead.